docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The date pickers are now part of the Advanced components
+description: XXXX
+date: 2022-04-03T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+---
+
+We're excited to announce that, after more than 18 month in the lab, the date picker components are now part of the X packages.
+
```

Comment:
I think this is worth explaining. Sources: [Date Picker](https://www.designencyclopedia.io/element/date-picker), [A hunt for the perfect date picker UI](https://uxdesign.cc/date-picker-design-5c5ef8f35286)

```suggestion

## What is a date picker?

A date picker is an interface control that enables the user to select a date from a menu.

Common design patterns include text inputs, dropdown lists, calendars, timelines, and scrolling pickers on mobile devices. 
The user may need to select an individual date, or a range of dates.

```